### PR TITLE
Update inboundSMS Documenation as per updated CiviCRM Core PR #10347

### PR DESCRIPTION
**Pull request by seamuslee001**

### CODE REVIEW

File: docs/hooks/hook_civicrm_inboundSMS.md [hunk 1]
```diff
@@ -39,20 +29,20 @@ Alter the incoming SMS From number to match how phone numbers are stored in the
 
 ```php
 <?php
-function myextension_civicrm_inboundSMS(&$from, &$fromContactID = NULL, &$to, &$toContactID = NULL, &$body, &$trackID) {
+function myextension_civicrm_inboundSMS(&$message) {
   // Alter the sender phone number to match the format used in database
-  $from = str_replace('+614', '04', $from);
+  $from = str_replace('+614', '04', $message->from);
```
Comment (ineffyble) [inline docs/hooks/hook_civicrm_inboundSMS.md]: This isn't going to alter the actual variable in $message->from is it?

File: docs/hooks/hook_civicrm_inboundSMS.md [hunk 2]
```diff
@@ -67,20 +57,21 @@ Send an automatic response to incoming messages
...
 Implement custom logic to match the message to the sender
 
 ```php
 <?php
-function myextension_civicrm_inboundSMS(&$from, &$fromContactID = NULL, &$to, &$toContactID = NULL, &$body, &$trackID) {
+function myextension_civicrm_inboundSMS(&$message) {
   // Implement custom matching logic
   // If there are multiple contacts with the phone number, preference the one that has been sent an SMS most recently
+  $escapedFrom = CRM_Utils_Type::escape($message->from, 'String');
   $fromContactID = CRM_Core_DAO::singleValueQuery("
```
Comment (ineffyble) [inline docs/hooks/hook_civicrm_inboundSMS.md]: Ditto for this one

Comment (seamuslee001) [inline docs/hooks/hook_civicrm_inboundSMS.md]: No ok and same as above

**Comment by seamuslee001** [comment]
@ineffyble Have updated examples now

**Pull request merged**